### DISCUSSION TITLE
Check G6 serial for spaces

### DISCUSSION THREAD

**Discussion by robertrub** [NightscoutFoundation/xDrip#2884] [Ideas]
Minor bug. If you enter a space by error after the G6 transmitter (if you copy and paste the serial per ex), xDrip will not connect to the transmitter and will not give any error messages. It took me some time to think of checking for a space character at the end...

It would be nice if xDrip could check the serial format when entered (or when quitting the parameters page).

**Comment by Navid200** [comment]
Can you check the xDrip logs?
Is there nothing related in the logs?

**Reply by robertrub** [reply]
I have 2 Anubis. I changed the transmitter serial of the main xDrip and inserted the serial that was in the variant and then put the serial of the main in the variant. Then stopped sensor on variant. All this was arouy 9AM - 9:30AM

This log is from the main (that couldn't connect because of the space character at the end of the serial)

![Screenshot_20230614_143647_xDrip+.jpg](https://github.com/NightscoutFoundation/xDrip/assets/76626474/45e95170-5a1e-43e0-8bc6-0c119fa470d8)

**Comment by Navid200** [comment]
This is the first time I am hearing this and sounds like a problem to me.
I need to do some tests to see if I can recreate this problem.  It may take a while.
Please create an issue.  

Please note I may find out that there is no issue and close it if I can show that this will not happen as is.
Alternatively, someone else may close it if they believe that this is an intentional behavior.

**Reply by robertrub** [reply]
If you have a G6, add a space to the serial and see if connects. I'll open an issue.

**Reply by robertrub** [reply]
I can't find where to open an issue in GitHub :( Is there another place for issues ?

**Reply by Navid200** [reply]
Please include a link to this discussions.
Here are issues: https://github.com/NightscoutFoundation/xDrip/issues
And please never open an issue unless we ask.  Thanks for that.

**Reply by robertrub** [reply]
Tnx. Issue opened.

https://github.com/NightscoutFoundation/xDrip/issues/2885
